rtp.c

```diff
@@ -1173,8 +1174,8 @@ gboolean janus_rtp_simulcasting_context_process_rtp(janus_rtp_simulcasting_conte
 					if(context->substream_target_temp < 0)
 						context->substream_target_temp = 0;
 					if(context->substream_target_temp != prev_target) {
-						JANUS_LOG(LOG_WARN, "No packet received on substream %d for a while, falling back to %d\n",
-							context->substream, context->substream_target_temp);
+						JANUS_LOG(LOG_WARN, "No packet received on substream %d for %d Milliseconds, falling back to %d\n",
```

Comment:
You may want `PRId64` instead of `%d` here

Reply:
Our code style is to use `%"SCNi64"` in those instances (assuming this is a `int64_t`?).

Reply:
Ah ok, that makes since given that internally `janus_vprintf` is calling `g_vasprintf`.

Reply:
Hi, Thanks. I've fixed it. Please also review the multistream PLI fix; it includes additional changes: https://github.com/meetecho/janus-gateway/pull/3423."